src/firewall-cmd.in

```diff
@@ -1152,6 +1146,20 @@ except FirewallError as msg:
     cmd.print_and_exit("Error: %s" % msg, code)
 
 fw.setExceptionHandler(cmd.exception_handler)
+
+if not (options_standalone or options_ipset or \
+        options_icmptype or options_service or options_helper or \
+        options_config or options_zone_ops or options_policy or \
+        options_direct or options_permanent_only):
+    state = fw.get_property("state")
+    if state == "RUNNING":
+        cmd.print_msg("State: running\n")
+    elif state == "FAILED":
+        cmd.print_msg("Failed\n")
+    else:
+        cmd.print_msg("State: not running\n")
+    cmd.fail(parser.format_usage() + "No options specified.\n\nFor usage information, use 'firewall-cmd --help' or firewall-cmd(1) man-page")
```

Comment:
We should be modifying `format_usage()` to use the improved usage message about `--help` and the man page.